gpbft/participant.go

```diff
@@ -485,7 +485,16 @@ func (p *Participant) fetchCommittee(instance uint64) (*committee, error) {
 		if err := power.Validate(); err != nil {
 			return nil, fmt.Errorf("instance %d: %w: invalid power: %w", instance, ErrValidationNoCommittee, err)
 		}
-		comt = &committee{power: power, beacon: beacon}
+
+		// TODO: filter out participants with no effective power after rounding?
+		// TODO: this is slow and under a lock, but we only want to do it once per
+		// instance... ideally we'd have a per-instance lock/once, but that probably isn't
+		// worth it.
```

Comment:
IMO, punt till later (probably not much worse than fetching the committee under the lock).